components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -1181,6 +1184,31 @@ public AccessTokenDO getAccessToken(String accessTokenIdentifier, boolean includ
         return dataDO;
     }
 
+    private Map<String, String> getAccessTokenExtendedAttributeParametersFromDB(String accessTokenIdentifier)
```

Comment:
```suggestion
    private Map<String, String> getAccessTokenExtendedAttributeParameters(String accessTokenIdentifier)
```

Reply:
Fixed in 475e57c84e607674b97ebd3efb0c0a67a6ac9f1c